pypose/__init__.py

```diff
@@ -14,3 +14,24 @@
 from .lietensor.function import *
 from . import module, optim
 from .basics import *
+
+
+def digit_version(version_str):
+    digits = []
+    for x in version_str.split('.'):
+        if x.isdigit():
+            digits.append(int(x))
+        elif x.find('rc') != -1:
+            patch_version = x.split('rc')
+            digits.append(int(patch_version[0]) - 1)
+            digits.append(int(patch_version[1]))
+    return digits
+
+
+torch_min_ver = '2.0'
+torch_version = digit_version(torch.__version__)
+
+
+assert digit_version(torch_min_ver) <= torch_version, \
```

Comment:
How about 
```
from packaging import version
assert version.parse('2.0') <= version.parse(torch.__version__)
```

Reply:
that works

Reply:
Also add `packaging` to runtime.txt?

Reply:
may not be necessary, our existing dependencies need it?

Reply:
Indeed not necessary, checked that the commit afterwards passes testing